client/src/pages/Roadmaps/Card.jsx

```diff
@@ -5,7 +5,7 @@ const Card = ({ bannerImage, id, title, tagline }) => {
   return (
     <Link to={`/roadmaps/${id}`}>
       <section className="roadmap-card">
-        <img src={bannerImage} alt="" />
+        <img src={bannerImage} alt="Roadmaps banner" />
```

Comment:
Use the title available through props as the title here.

Reply:
I have added the alt text from the `title` prop. Thank you.